index.js

```diff
@@ -299,7 +299,7 @@ class CSGOCdn extends EventEmitter {
      * Loads the CSGO dir VPK specified in the config
      */
     loadVPK() {
-        this.vpkDir = new vpk(this.config.directory + '/pak01_dir.vpk');
+        this.vpkDir = new vpk(this.config.directory + '/platform_pak01_dir.vpk');
```

Comment:
Why it was replaced?

If I use this code, I get the following error:

```
(node:40861) UnhandledPromiseRejectionWarning: Error: ENOENT: no such file or directory, open 'data/platform_pak01_dir.vpk'
```

In the old version, everything is fine:

```suggestion
        this.vpkDir = new vpk(this.config.directory + '/pak01_dir.vpk');
```

Reply:
You're right. I had a custom script running locally which for some reason renamed it